### DISCUSSION TITLE
Ajouter la caisse de retraite des vétérinaire (CARPV)

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2683] [Nouveaux simulateurs]
Pour nos amis les amis des animaux.
-> https://www.carpv.fr/

**Comment by wiinxt** [comment]
> Bonjour, dans le simulateur de revenu pour les professions libérales, je ne trouve aucune case qui correspond à ma profession (vétérinaire libérale). Peut-être ai-je mal cherché? pouvez-vous m'aider?
